components/specification/src/ome/specification/OmeValidator.java

```diff
@@ -233,13 +248,20 @@ public Document parseFileWithStreamArrayToSdtErr(File file, StreamSource[] schem
         SchemaResolver theTestClassResolver = new SchemaResolver();
         sFactory.setResourceResolver(theTestClassResolver);
 
-        Schema theSchema = sFactory.newSchema( schemaStreamArray );
-
-        // Version - one step parse and validate (print error to stdErr)
-        dbf.setSchema(theSchema);
+        Schema theSchema = sFactory.newSchema(schemaStreamArray);
+        // Version - two step parse then validate (throws error as exception)
```

Comment:
I assume this change in error handling is okay; does it need mentioning in release notes?